### PR TITLE
Slightly fixes in documentation and zip scan

### PR DESCRIPTION
**Pull request by babenek**
## Description
- slight comments fix

## How has this been tested?

- [x] CI

### CODE REVIEW

File: credsweeper/deep_scanner/pdf_scanner.py [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
-    """Realises pdf scanning"""
+    """Implements pdf scanning"""
 
     def data_scan(
             self,  #
             data_provider: DataContentProvider,  #
             depth: int,  #
             recursive_limit_size: int) -> List[Candidate]:
-        """Extracts text from PDF elements and whole text, then launch data_scan"""
+        """Tries to scan PDF elements recursive and whole text on page as strings"""
```
Comment (kmnls) [inline credsweeper/deep_scanner/pdf_scanner.py:25]:
```suggestion
        """Tries to scan PDF elements recursively and whole text on page as strings"""
```

**Review by kmnls — APPROVED** [review]

File: credsweeper/file_handler/file_path_extractor.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def check_file_size(
             config: Config,  #
             reference: Union[str, Path, io.BytesIO, Tuple[Union[str, Path], io.BytesIO]]) -> bool:
         """
-        Checks whether the file is oversize limit
+        Checks whether the file is over the limit from configuration
```
Comment (csh519) [inline credsweeper/file_handler/file_path_extractor.py:147]:
```suggestion
        Checks whether the file is over the size limit from configuration
```

**Review by csh519 — COMMENTED** [review]
Please check benchmark action failure too.

**Review by kmnls — APPROVED** [review]

**Review by csh519 — APPROVED** [review]
Thank you for fix!

LGTM :+1:

**Pull request merged**